{{cookiecutter.project_slug}}/setup.py

```diff
@@ -18,7 +18,7 @@
     classifiers=["Programming Language :: Python"],
     author="{{cookiecutter.author_name}}",
     author_email="{{cookiecutter.author_email}}",
-    url="",
+    url="https://github.com/marcosschroh/cookiecutter-faust",
```

Comment:
was this intended?

Reply:
Oh, I see you fixed it in https://github.com/marcosschroh/cookiecutter-faust/pull/23/files